### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by teamotitan**
Quick Start was catered towards Address Book

This does not really match our product which is UGTeach

Let's
* alter the User Guide to suit our product better

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -17,15 +17,18 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14a-1/tp/releases).
```
Comment (Feng1231) [inline docs/UserGuide.md:20]: Thanks for catching this 😄

**Review by Feng1231 — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 2]
```diff
@@ -17,15 +17,18 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-   ![Ui](images/Ui.png)
+1. Open a command terminal, `cd` into the folder that you put the jar file in
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+1. Use the `java -jar ugteach.jar` command to run the application.<br><br>
+   A GUI similar to the image shown below should appear in a few seconds. Note how the app contains some sample data.
+   <br>
+   ![Ui](images/UGTeach.png)
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br><br>
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:31]: Might consider fixing the indexing

Comment (teamotitan) [reply docs/UserGuide.md:31]: I'm not very sure, but on markdown the indexing looks okay

Comment (Soumyadip-cmd) [reply docs/UserGuide.md:31]: Yup but abit funky so might be better to change.

Comment (teamotitan) [reply docs/UserGuide.md:31]: You have a good point. I will change it this instant!

**Pull request merged**